.gitbook.yaml

```diff
@@ -4,5 +4,12 @@ structure:
   readme: introduction.md
   summary: toc.md
 
-#redirects:
-#  help: ./support.md
\ No newline at end of file
+redirects:
+ reference/migration-guide/README.md: how-to/manage-the-zenml-server/migration-guide/migration-guide.md
+ reference/migration-guide/migration-zero-twenty.md: how-to/manage-the-zenml-server/migration-guide/migration-zero-twenty.md
+ reference/migration-guide/migration-zero-thirty.md: how-to/manage-the-zenml-server/migration-guide/migration-zero-thirty.md
+ reference/migration-guide/migration-zero-forty.md: how-to/manage-the-zenml-server/migration-guide/migration-zero-forty.md
```

Comment:
After the md needed on the left side or right side? Not sure.. can you check if  this works by syncing to a Gitbook space